proposals/stage-0/TRAVEL_FUND/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenJS Travel Fund
+>  Stage 0
+
+Tracked by https://github.com/openjs-foundation/cross-project-council/issues/172
+
+## Champion
+
+Jonah Stiennon @jonahss
+
+## Description
+
+The Node.js organizations that existed prior to the formation of the OpenJS foundation oversaw a monetary fund for the purpose of member and affiliate travel to, and lodging at, events related to the foundation's mission. This proposal proposes moving the management responsibilities of this fund to the Cross Project Council (CPC), and making the funds available to all members of this broader organization. Since all projects have voting representation on the CPC, the beneficiaries of the fund will jointly manage it, and the previous owners of the fund will still vote on its use.
+
+We shall review the process in this proposal, but the intent is to leave the mechanics of the fund unchanged, simply moving from the purview of the Node.js project into the domain of the OpenJS Foundation, overseen by the CPC. This operation can be viewed as a refactor which moves the travel fund up one level in the organizational tree.
+
+The current fund is used mostly for attending collaborator summits, and we do not expect this to change.
+
+## Difference from the current process
+
+The existing process requires approval from two members of the Node.js TSC and two members of the Node.js Community Committee.
+The new process will require approval from four members of the CPC.
+
+The role of fund "treasurer" will be removed, as it has been found to be redundant and is not currently filled (see https://github.com/openjs-foundation/cross-project-council/pull/268#issuecomment-513888283).
+
+## Practical Specifics
+
+- Move relevant files from nodejs/admin to openjs-foundation/cross-project-council such as:
+  - https://github.com/nodejs/admin/blob/master/MEMBER_TRAVEL_FUND.md
+  - https://github.com/nodejs/admin/tree/master/TravelFunds
+  - https://github.com/nodejs/admin/blob/master/travel-visas.md
+  - https://github.com/nodejs/admin/blob/master/expense-report-template.xls
+  - https://github.com/nodejs/admin/blob/master/reimbursement_process.pdf
+- Modify these files to account for OpenJS rather than Node.js organizations
+- Remove mentions from nodejs/admin README, add links to CPC README
+- Remove the role of treasurer
+
+## Required Resources
+
+@brianwarner reports that everything can stay the same for the tools and accounts involved.
+No other resources required.
+
+## How would success be measured?
+
+Success will be achieved if Node.js and non-Node.js members use the travel fund to attend a Foundation-related event, and nobody notices the difference.
+
+## Why this proposal is important
+
+Organizationally, the Node.js project is at the same level as the other impact projects in the Foundation, and so the travel fund should benefit all projects equally in opportunity. Plus, the sponsorships which pay for the fund sponsor the entire Foundation, not just the Node.js project.
```

Comment:
One of the assumption here is that the Node.js project should not receive any less funding due to the merger. I think this should be clarified.

cc @Trott.

Reply:
I think that that depends more upon the board's discretion with the budget, but I can add language to suggest that the budget be raised to accommodate the increase in people asking for funding.

Reply:
> One of the assumptions here is that the Node.js project should not receive any less funding due to the merger.

How could we ensure this if there is a common fund and approvals are done by CPC members?
Or is your clarification more around explaining what we'll do so that it will mostly likely be the case (since for example we've expanded the size of the fund so that we expect that all request from members of the Node.js project in addition to the request from members of other projects can be accomodated)

Reply:
>Or is your clarification more around explaining what we'll do so that it will mostly likely be the case (since for example we've expanded the size of the fund so that we expect that all requests from members of the Node.js project in addition to the request from members of other projects can be accomodated)

Yes, that is what I meant. I'd like to avoid codifying specific quotas and rules unless we actually need them. Since it doesn't even appear that there has ever been an issue of abuse of this fund, or lack of budget.

Should we count the number of new members being added to the fund due to the foundation merge, calculate the percentage increase, and get pre-approval of an expanded budget before continuing?

Reply:
> Should we count the number of new members being added to the fund due to the foundation merge, calculate the percentage increase, and get pre-approval of an expanded budget before continuing?

I think so. The overall expectation was that Node.js would not suffer from the merger.

Reply:
It seems a rough estimate of the group sizes is worth it. I'll collect some data.

Reply:
@mcollina Do you think we could make that a part of stage-1 requirements, and be able to move this to stage-0 without it?

Reply:
> Do you think we could make that a part of stage-1 requirements, and be able to move this to stage-0 without it?

We are not in agreement with the wording of this paragraph, and I'm not LGTM this even for Stage-0. This paragraph should state that given we have (in theory!) a larger pot of money to spend, the travel fund could be expanded to cover all projects. This was the overall intent of the merger, and the wording of this paragraph goes against it as the fund should benefit "not just the Node.js project". 

(As an individual on the TSC, I would put a -1 on a proposal with this text. I'll bring this with the rest of the TSC. My reading of this is to remove the travel fund from Node.js in a way that will definitely not benefit the project.)

I'm happy for the Board consideration to go into the "What is necessary to complete this proposal" section.

@mhdawson I hope this answer your question as well, if not let me now.

Reply:
@mcollina 

Can we further discuss the wording you'd prefer to see here? I had a little trouble understanding your latest comment.

>My reading of this is to remove the travel fund from Node.js in a way that will definitely not benefit the project.

The purpose of the CPC is to benefit all the projects.
My goal with this proposal is to keep things the same for the Node.js project, but _extend_ the benefits to the other projects. Could you be satisfied by some future research where I provide numbers for current usage trends, and an increased budget to preserve the current benefits to the Node.js members, while extending the benefits to all projects?

This year, I am given to understand that some of the non-Node.js project members were awarded travel funds to attend the collaborator summit in Berlin. Do you feel that this caused a negative impact to the Node.js project?

-------------------------------------------------------

Board approval is necessary to complete the proposal. I will add that to the text now.

Reply:
I suggest changing
>Organizationally, the Node.js project is at the same level as the other impact projects in the Foundation, and so the travel fund should benefit all projects equally in opportunity. Plus, the sponsorships which pay for the fund sponsor the entire Foundation, not just the Node.js project.

with:
> The Node.js project has demonstrated the value of providing a travel fund which allows project members to get together at collaborator summits and to represent/advocate for the project at other events. It is important to expand the travel fund (both resource and usage) so that other projects can benefit as well.

Reply:
@mcollina is that along the lines of what you were thinking?

Reply:
Yes exactly, thanks.

Reply:
Sounds great! Made the change suggested.